### PR TITLE
fix(version): delete version

### PR DESCRIPTION
**Pull request by aeng**
https://zanata.atlassian.net/browse/ZNTA-953

### CODE REVIEW

File: zanata-war/src/main/webapp/WEB-INF/layout/project/languages-list.xhtml [hunk 1]
```diff
@@ -70,7 +70,9 @@
   <script type="text/javascript">
   //<![CDATA[
     // refresh checkbox facades on language lists
-    zanata.form.appendCheckboxes(jQuery('##{id}').element);
+    if (typeof zanata != 'undefined') {
```
Comment (carlosmunoz) [inline zanata-war/src/main/webapp/WEB-INF/layout/project/languages-list.xhtml]: why would this happen? and is it ok to ignore it?

Comment (aeng) [inline zanata-war/src/main/webapp/WEB-INF/layout/project/languages-list.xhtml]: `zanata.form.appendCheckboxes` only needed after the first load, which `zanata` will be available by then.

**Comment by carlosmunoz** [comment]
:+1:

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested

**Pull request merged**